### ISSUE TITLE
table inside the attachment not showing button for Install Map

### ISSUE THREAD

**Issue by darkplataform** [bitrise-io/steps-slack-message#38]
Good day ,

The table inside the attachment is not showing button for Install Map, the env variable BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP was populated correctly in a previous step "Deploy to bitrise"

This is my configuration for the slack step for the  list of buttons:
![image](https://user-images.githubusercontent.com/5014699/38396365-34eff89a-38fd-11e8-89ad-1e78891818a0.png)

You probably noticed that I also put the BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP  in the table inside the attachments, I did this on purpose because I just figure out that if I put this I get the contents of this variable as a  BLOB text.  But this is totally non user/QA/tester friendly

This is my Build URL: https://www.bitrise.io/build/c55b1d0497d39346 as requested in [https://github.com/bitrise-io/steps-deploy-to-bitrise-io/issues/59](url)

**Comment by gitizenme** [comment]
👍 to see this work correctly

**Comment by lszucs** [comment]
Hi @darkplataform,

We released a new feature, which gives you the opportunity to format the map value using a Go template. The `BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP` variable will be populated with the output of the template.

You may provide the template in the `public_install_page_url_map_format` input.

For example, **to get nice button links in the Slack message**:

```
- public_install_page_url_map_format: |
    {{range $index, $element := .}}
        {{$element.File}}|{{$element.URL}}
    {{end}}
```

Let me know if this helps and if we can close the issue.

**Comment by RyanNewsom** [comment]
> Hi @darkplataform,
> 
> We released a new feature, which gives you the opportunity to format the map value using a Go template. The `BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP` variable will be populated with the output of the template.
> 
> You may provide the template in the `public_install_page_url_map_format` input.
> 
> For example, **to get nice button links in the Slack message**:
> 
> ```
> - public_install_page_url_map_format: |
>     {{range $index, $element := .}}
>         {{$element.File}}|{{$element.URL}}
>     {{end}}
> ```
> 
> then, in your Slack step, you would provide the input as follows:
> 
> ```
> - buttons: |
>     ${BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP}
> ```
> 
> Note: you do not provide a pipe separated value - the formatting has been done by the previous Go template already!
> 
> Let me know if this helps and if we can close the issue.

I don't think this works anymore. When I paste in this button set-up and save, leave the yml edit tab, and come back to it, it always reformats it to this, and I'm getting 0 buttons shown.
- buttons: "${BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP}"
